pages/interface/components/ReadTime/index.js

```diff
@@ -0,0 +1,14 @@
+import { Text } from '@/TabNewsUI';
+
+function getReadTime(text, wpm = 260) {
```

Comment:
Será que vale deixar o wpm mutável? 

Talvez sim, mas tentei pensar aqui no caso de uso que alteraríamos e não encontrei, talvez tornar ele uma const, não sei, o que acha?

```suggestion
const WORDS_PER_MINUTE = 260

function getReadTime(text) {
```